mods/arcade/lib/arcade-main/templates/arcade-container.template.js

```diff
@@ -1,6 +1,11 @@
 module.exports = ArcadeContainerTemplate = (app, mod) => {
   return `
     <div class="header-shim" style=""></div>
-    <div id="arcade-container" class="arcade-container"></div>
+    <div id="arcade-container" class="arcade-container">
+      <div class="page-header">
+        <h1 class="page-title">Arcade</h1>
+        <p class="page-subtitle">This is a tech demo for Saito. Play games and win cryptos!</p>
```

Comment:
I don't think we want to say "win cryptos" here because it's not true and we don't want to brand as a casino...

I'd have said "play games securely on the blockchain!", but let's run this past David to see if maybe he has some better idea.